src/tasks/sep/034-op-ink-sep-fusaka-prestate/config.toml

```diff
@@ -0,0 +1,37 @@
+templateName = "OPCMUpdatePrestateV410"
+
+[[l2chains]]
+chainId = 11155420
+name = "OP Sepolia Testnet"
+
+[[l2chains]]
+chainId = 763373
+name = "Ink Sepolia Testnet"
+
+[[opcmUpgrades]]
+chainId = 11155420
+absolutePrestate = "0x03eb07101fbdeaf3f04d9fb76526362c1eea2824e4c6e970bdb19675b72e4fc9"  # op-program/v1.6.1 - https://github.com/ethereum-optimism/superchain-registry/blob/47f21fb1291999cb9a46653dd92d7cd55b6e3821/validation/standard/standard-prestates.toml#L6C9-L6C75
+# The above prestate needs to be updated with the correct Fusaka Absolute Prestate
```

Comment:
I'm not sure what our normal process is, but it seems like it would be better to follow-up with a PR that includes this task with the correct data.